v0.1/core/visualizationParameters.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/core/visualizationParameters.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Definition of visualization parameters",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type",
+    "@id",
+    "bestViewPoint",
+    "definedIn",
+    "internalFileIdentifier"
+  ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/VisualizationParameters"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier (id).",
+      "pattern": "sands/v0.1/core/visualizationParameters/([A-Za-z0-9_-]+).json"
+    },
+    "bestViewPoint": {
+      "$ref": "https://schema.hbp.eu/sands/core/coordinate.schema.json",
```

Comment:
Needs to point to definitions file.

Reply:
:see_no_evil: these are copy paste errors... I'll correct it